docs/getting-started/welcome.md

```diff
@@ -0,0 +1,100 @@
+---
+sidebar_label: '👉 Start Here'
+description: 'Start here as a developer to get the basics and start coding fast.'
```

Comment:
Suggest making the description more SEO oriented with more keywords like "LUKSO", "dApp", "blockchain", etc...